Cargo.toml

```diff
@@ -79,3 +79,12 @@ extism = { version = "1.2.0", optional = true }
 mithril-client = { version = "^0.8", optional = true, features = ["fs"] }
 miette = { version = "7.2.0", features = ["fancy"] }
 itertools = "0.12.1"
+tungstenite = "0.24.0"
+tokio-tungstenite = "0.24.0"
```

Comment:
would it be possible to hide these two dependencies under an `hydra` feature?

Oura's plugin nature makes it easy to bloat the dependency graph very quickly. When try to hide all plugin-specific dependencies under a its own flag.

Reply:
Done: 3ab7287e7ea1548c38e75e2adc4bee9e855ecfc1

Not sure what the right approach is with the new dev-dependencies, or if they are fine as-is from being dev-dependencies, so left them as-is.